tensorrt_llm/_utils.py

```diff
@@ -1063,3 +1063,14 @@ def _unique_tokens_to_json(data):
             "token_id": data.token_id,
             "token_extra_id": data.token_extra_id
         }
+
+
+def is_multi_device_enable():
+    """
+    This method evaluates if we are running on multiple GPUs and the flag ENABLE_MULTI_DEVICE is set.
+    So we can avoid broadcast calls on single GPU.
+    Issue: https://github.com/NVIDIA/TensorRT-LLM/issues/5927
+    ENABLE_MULTI_DEVICE is true by default when building tensorrt-llm so we need to also check
+    the number of devices
+    """
+    return local_comm.Get_size() > 1 and ENABLE_MULTI_DEVICE
```

Comment:
is `local_comm` the communicator for the  node? just making sure this won't break the multi-node, but 1 GPU per node case